routes/events/index.js

```diff
@@ -198,6 +227,11 @@
       }
     });
 
+    function extractPlaceId(event) {
+      const placeId = event.place["@id"].split("/");
+      return placeId[5];
```

Comment:
This might be unstable, strip last '/' before split and always use the last segment of the path